cmake/modules/Assets.cmake

```diff
@@ -4,33 +4,36 @@ function(add_assets_target)
     cmake_parse_arguments(
         _ASSETS
         ""
-        "TARGET;SOURCE_DIR;DEST_DIR;DEVEL"
-        ""
+        "TARGET;SOURCE_DIR;DEST_DIR;DEVEL;"
+        "DEPENDS"
         ${ARGN}
     )
 
     if(NOT ${_ASSETS_DEVEL})
         set(EXCLUDED --exclude \"*-devel*\")
     endif()
+    if(NOT ${PROJECT_TARGET} STREQUAL "TARGET_Linux")
+        set(COPY_LUTS ${_ASSETS_SOURCE_DIR}/Luts.bin)
+    endif()
 
     add_custom_target(
         ${_ASSETS_TARGET}
-        DEPENDS json-target
+        DEPENDS ${_ASSETS_DEPENDS}
         COMMAND mkdir -p ${_ASSETS_DEST_DIR}
         COMMAND rsync -qravu --delete
             ${_ASSETS_SOURCE_DIR}/.boot.json*
             ${_ASSETS_SOURCE_DIR}/personalization.json
             ${_ASSETS_DEST_DIR}
-        COMMAND rsync -qravu --delete
+        COMMAND rsync -qravu
```

Comment:
why without deleting?

Reply:
this is because of what this target actually does is copying stuff from `image` catalog in `proper` places
now we have target which copies downloaded elements directly to build catalog and its set as a dependency to this `asset` target to keep it like that:
```
download & generation of assets  <- assets <- creation of image
```

but if `assets` target removes `assets` directory where we already put data so that it would end up in image we won't have it in it - without any error risen... 
now you would probably as "Ok, then why don't we copy assets to `image` catalog as we did.
This is because we can't do it. If we did - we won't have properly built packages, as we are copying whole directories here - we will copy all files: then image generated for tests will have both assets for tests and assets for i.e. bell and we do not want that.

At best I would rather remove all of that rsync stuff, then generate different `asset` targes (one for tests, one for bell, one for pure one for pure with developer data) just aggregating its dependencies. This way we wouldn't have to i.e. have hacks in line 35 or weird `if's` for LUTS - as different configurations would result in a different set of data copied.. That would be out of scope here as I do not want to refactor our cmake code too much.